From 2e7d2e22935950ece32ff3acd6837c79800752dd Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 11 Mar 2013 13:21:14 -0400
Subject: [PATCH] fix DESTDIR handling with udev rules install again

Commit 102e29d16c3fd9b414eeac53c794302f902ae8da reverted the DESTDIR
logic when upgrading the udev path handling.

It also slightly broke the mkdir setup by not creating the rules.d
subdir under the udev path.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 SConstruct | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/SConstruct b/SConstruct
index a56fa9b..1c187f3 100644
--- a/SConstruct
+++ b/SConstruct
@@ -1662,10 +1662,10 @@ if env['python']:
 # is plugged in.
 
 Utility('udev-install', 'install', [
-    'mkdir -p ' + env['udevdir'],
-    'cp $SRCDIR/gpsd.rules ' + env['udevdir'] + '/rules.d/25-gpsd.rules',
-    'cp $SRCDIR/gpsd.hotplug ' + env['udevdir'],
-    'chmod a+x ' + env['udevdir'] + '/gpsd.hotplug',
+    'mkdir -p ' + DESTDIR + env['udevdir'] + '/rules.d',
+    'cp $SRCDIR/gpsd.rules ' + DESTDIR + env['udevdir'] + '/rules.d/25-gpsd.rules',
+    'cp $SRCDIR/gpsd.hotplug ' + DESTDIR + env['udevdir'],
+    'chmod a+x ' + DESTDIR + env['udevdir'] + '/gpsd.hotplug',
         ])
 
 Utility('udev-uninstall', '', [
-- 
1.8.1.2

